src/popover/popover.ts

```diff
@@ -115,6 +115,10 @@ export class NgbPopover implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() container: string;
+  /**
+   * A flag indicating if a given popover is disabled and should not be displayed.
+   */
+  @Input() disabled = false;
```

Comment:
1. Don't think it will be acceptable, too many collisions, ex:

```html
<button [disabled]="true" ngbPopover>...</button>
```

maybe `[showPopover]` or `[disablePopover]`?

```html
<div [ngbPopover]="text" [showPopover]="text !== ''" />
<div [ngbPopover]="text" [disablePopover]="text === ''" />
```

2. All other inputs initial values come from `NgbPopoverConfig`. I think we should put this one there as well. Not 100% sure though if there is a use case for massively initializing all popovers as disabled... So this is questionable